playwright.config.ts

```diff
@@ -69,8 +69,7 @@ const config = defineConfig<TestOptions>({
     },
   ]),
   forbidOnly: !!process.env.CI,
-  // no parallelization, otherwise the `waku` command will have race conditions
-  workers: 1,
+  workers: process.env.CI ? 1 : 3,
```

Comment:
playwright recommend use worker 1 for stable runner, I will only run for local dev